test/interceptors/response.test.ts

```diff
@@ -361,4 +361,33 @@ describe('Response Interceptor', () => {
     assert.equal(storage.state, 'cached');
     assert.equal(storage.data?.data, true);
   });
+
+  // https://github.com/arthurfiorette/axios-cache-interceptor/issues/922
+  it('Aborted requests do not clear its cache afterwards if previous request was cached', async () => {
+    const axios = mockAxios();
+
+    const id = '1';
+
+    // First request
+    await axios.get('url', { id });
+
+    // Second request cancelled
+    const controller = new AbortController();
+    const cancelled = axios.get('url', { id, signal: controller.signal });
+    controller.abort();
```

Comment:
since the usage of `mockAxios()`, this here only tests when a request is cancelled before being dispatched to network, could you chance to a normal axios test instance? just like tests at `test/interceptors/stale-if-error.test.ts`

Reply:
maybe you could extend the `Cancelled deferred still should save cache after new response` unit test instead of creating a new one, this way both pre and post cancellation will be tested using an actual connection.

Reply:
I'm not sure how I can use a real axios instance. The first request has to work, to cache an actual answer

Reply:
you can pre populate it directly using `axios.storage`.

Reply:
updated